orchagent/pfcwdorch.cpp

```diff
@@ -39,6 +39,42 @@ PfcWdOrch<DropHandler, ForwardHandler>::PfcWdOrch(DBConnector *db, vector<string
     timer->start();
 }
 
+template <typename DropHandler, typename ForwardHandler>
+QueueMcDropCounters PfcWdOrch<DropHandler, ForwardHandler>::getQueueMcDropCounters(
+        const Port& port,
+        uint8_t pfcMask)
+{
+    SWSS_LOG_ENTER();
+
+    vector<FieldValueTuple> fieldValues;
+    QueueMcDropCounters counters;
+    counters.fill(numeric_limits<uint64_t>::max());
+
+    for (uint8_t prio = 0; prio < PFC_WD_TC_MAX; prio++)
+    {
+        sai_object_id_t queueId = port.m_queue_ids[prio + PFC_WD_TC_MAX];
+
+        if (!m_countersTable->get(sai_serialize_object_id(queueId), fieldValues))
+        {
+            continue;
+        }
+
+        for (const auto& fv : fieldValues)
+        {
+            const auto field = fvField(fv);
+            const auto value = fvValue(fv);
+
+
+            if (field == "SAI_QUEUE_STAT_PACKETS")
```

Comment:
Maybe I missed something, the function called getQueueMcDropCounters but we are trying to get SAI_QUEUE_STAT_PACKETS? Can you help me understand what we are trying to do?

Reply:
AFAIK, the agreement was that SONiC does not apply any configuration to make packets drop on lossless MC queues. Its done internally bu SAI, so SONiC still reads SAI_QUEUE_STAT_PACKETS to see how much packets were in that queue.

Reply:
Sorry I still cannot get the idea why we query the SAI_QUEUE_STAT_PACKETS but name it as dropped packet?